purchase_incoterm/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Purchase Incoterm',
```

Comment:
I would rename the addon `purchase_partner_incoterm` to be inline with the existing [`sale_partner_incoterm`](https://github.com/OCA/sale-workflow/tree/14.0/sale_partner_incoterm).
+ Update the descriptions/user labels as well to be coherent:
```suggestion
    "name": "Default purchase incoterm per partner",
    "summary": "Set the supplier preferred incoterm on each purchase order",
```

Reply:
Thanks for reivew, branch updated

Reply:
I wonder it the name in the manifest should not be the name of the module without the underscore ?